docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -9,7 +9,7 @@ import MoreHelp from "/src/components/MoreHelp"
 # Sample CI configurations
 
 The sample configuration files below
-run [Semgrep CI](https://github.com/returntocorp/semgrep-action)
+run [Semgrep CI](https://github.com/returntocorp/semgrep-action) TODO
```

Comment:
@underyx should we just point to https://github.com/returntocorp/semgrep now?

Reply:
I think we should link to the docs for `semgrep ci` instead.

Reply:
Maybe there does not need to be any link actually... it may be just a distraction for the reader. I mean... there used to be a link to the Semgrep action repo - https://github.com/returntocorp/semgrep-action - but as I understand it now, the Semgrep CI now lives inside of the https://github.com/returntocorp/semgrep  repo, right? :-) @underyx

Reply:
Well I think it's useful to link a page that explains how they can configure the command in the CI jobs further.

Reply:
We can explain that this is why wanna link them to a different page, maybe in an info box.

> To see what settings you can configure for this job, please go to the `semgrep ci` documentation.

Reply:
So you mean to link them here maybe? :-) https://semgrep.dev/docs/semgrep-ci/configuration-reference/ @underyx I mean... I need to know where exactly we want readers to go. :-)

Reply:
I'm not sure exactly how the docs are getting reorganized, but we should have a page, nested under Semgrep CLI, that explains what the `semgrep ci` subcommand is. I'd like to link to that page.

Reply:
@underyx understand! Do you want to write a draft of such a document? I think that we can include it as a subsection here: https://semgrep.dev/docs/cli-usage/

Reply:
Should not the `semgrep ci` subcommand be a part `semgrep --help`?

Reply:
> @underyx understand! Do you want to write a draft of such a document? I think that we can include it as a subsection here: https://semgrep.dev/docs/cli-usage/

The content would be basically the same as the Semgrep CI page: https://semgrep.dev/docs/semgrep-ci/overview/ — `semgrep ci` has all the same selling points, and features. So maybe you could just reorganize this page and make sure it's clear this is all talking about a `semgrep` subcommand?

> Should not the semgrep ci subcommand description be a part of semgrep --help?

Yep, we're adding it there! But it'll only get one line of description, like this

```
Subcommands:
    ci - Run Semgrep in CI, can upload findings to semgrep.dev
```

Reply:
@underyx Adding a note to the overview of Semgrep CI: 

```
**Note**: Semgrep CI is a subcommand of Semgrep CLI, so you can test the code and behavior of Semgrep CI in your command line using `semgrep ci` command. This means that all the capabilities of Semgrep CLI are available in Semgrep CI.
```